Makefile

```diff
@@ -9,6 +9,12 @@ help: ## display this help message
 # Define PIP_COMPILE_OPTS=-v to get more information during make upgrade.
 PIP_COMPILE = pip-compile --rebuild --upgrade $(PIP_COMPILE_OPTS)
 
+TEMPLATES=$(wildcard cookiecutter-*)
```

Comment:
This auto-detects all directories with the `cookiecutter-` prefix, so we don't have to maintain this list when new ones are added :)